src/node.cc

```diff
@@ -3526,6 +3531,8 @@ static void PrintHelp() {
          "                             note: linked-in ICU data is present\n"
 #endif
          "  --preserve-symlinks        preserve symbolic links when resolving\n"
+         "  --preserve-symlinks-main   preserve symbolic links when resolving "
```

Comment:
this should include a `\n` after resolving and the next line should be indented to align.